crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -337,12 +337,12 @@ impl<T: StateRead + ?Sized> StateReadExt for T {}
 
 #[async_trait]
 pub(crate) trait StateWriteExt: StateWrite {
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(%address, %rollup_id))]
```

Comment:
Is it useful to have this information?

Reply:
Removed in [1ceff06](https://github.com/astriaorg/astria/pull/1255/commits/1ceff067db6646e6b61f07a31b86635c70af0d2d).